fda-model/metrics/Results.cs

```diff
@@ -20,10 +21,23 @@ private bool IsEADConverged()
         }
         private bool IsPerformanceConverged()
         {
-            //dont like this.
+            
+            List<bool> convergedList = new List<bool>();
+            //dont like this
             foreach (var key in PerformanceByThresholds.ThresholdsDictionary)
             {
-                return PerformanceByThresholds.ThresholdsDictionary[key.Key].ProjectPerformanceResults.ConditionalNonExceedanceProbabilityIsConverged();
+                convergedList.Add(PerformanceByThresholds.ThresholdsDictionary[key.Key].ProjectPerformanceResults.ConditionalNonExceedanceProbabilityIsConverged());
+            }
+            foreach (var convergenceResult in convergedList)
+            {
+                if (convergenceResult)
```

Comment:
if(!coverged) might be clearer. no skipped case.

Reply:
I've read in Clean Code to avoid the ! modifier in if statements for clarity, and I agree. I did change the name for clarity.